src/workflows/airqo_etl_utils/data_validator.py

```
@@ -271,7 +271,31 @@ def process_data_for_api(data: pd.DataFrame) -> list:
                 restructured_data.append(row_data)
 
             except Exception as ex:
-                traceback.print_exc()
+                logger.exception(f"Error ocurred: {e}")
```

Ruff

Undefined name `e`

(F821)